sys/libkern/bcopy.c

```diff
@@ -141,25 +150,41 @@ memcpy(void *dst0, const void *src0, size_t length)
 			TLOOP1(*--dst = *--src);
 		}
 		t = length / wsize;
-		TLOOP(src -= wsize; dst -= wsize;
-		    *(word *)dst = *(const word *)src);
+#if __has_feature(capabilities)
+		if (!keeptags) {
+			TLOOP(src -= wsize; dst -= wsize;
+			    *(word *)dst = (word)cheri_cleartag(
+			        (void * __capability)*(const word *)src));
+		} else
+#endif
+			TLOOP(src -= wsize; dst -= wsize;
+			    *(word *)dst = *(const word *)src);
 		t = length & wmask;
 		TLOOP(*--dst = *--src);
 	}
 done:
 	return (dst0);
 }
 
+void *
+memcpy(void *dst0, const void *src0, size_t length)
+{
+	return _memcpy(dst0, src0, length, true);
+}
+
 __strong_reference(memcpy, memmove);
 
 void
-(bcopy)(const void *src0, void *dst0, size_t length)
+bcopy(const void *src0, void *dst0, size_t length)
```

Comment:
I'm a little surprised this works given the use of builtins in the default defines.

Reply:
Didn't seem to cause issues but I'll revert it back to how it was.